osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/security/RsaEncryptionService.java

```diff
@@ -310,6 +317,36 @@ public static PrivateKey readPrivateKeyFromFile(final String filename) {
         }
     }
 
+    public static PrivateKey readPrivateKeyFromResource(final Resource resource) {
```

Comment:
Should this be public? Though at the same time: can we reuse this? (I have seen the same logic in for instance he-ui).

Reply:
it can be private indeed. will change that.

Reply:
> Should this be public? Though at the same time: can we reuse this? (I have seen the same logic in for instance he-ui).

I actually removed the exact copy of this Class from he-ui (yes it was copied 100%) and I now use this bean correctly.